### PR TITLE
[RFR] More test + Fixes

### PR DESCRIPTION
**Pull request by gshefer**
{{pytest: cfme/tests/containers/test_relationships_tables.py -v --use-provider cm-env1 }}

### CODE REVIEW

**Comment by gshefer** [comment]
@dajoRH please merge

**Comment by dajoRH** [comment]
Response to comment #277022994:

I am sorry but I cannot merge this PR for you. Please, address the issues pointed out in the lint comment and then try again.

**Comment by gshefer** [comment]
@dajoRH please merge

**Comment by dajoRH** [comment]
Response to comment #277517002:

I am sorry but I cannot merge this PR for you. Please, address the issues pointed out in the lint comment and then try again.

File: cfme/tests/containers/test_relationships_tables.py [hunk 1]
```diff
@@ -23,44 +23,43 @@
...
-                DataSet(Service, service_match_page),
-                DataSet(Node, node_match_page),
-                DataSet(Replicator, replicator_match_page),
-                DataSet(Image, image_match_page),
-                DataSet(Project, project_match_page)]
-rel_values = (0, 'Unknown image source', 'registry.access.redhat.com')
+TEST_OBJECTS = version.pick({
+    version.LOWEST: [Pod, Service, Node, Replicator, Image, Project],
+    '5.7': [Pod, Service, Node, Replicator, Image, Project, Template]
+})
```
Comment (mfalesni) [inline cfme/tests/containers/test_relationships_tables.py]: **REQUIRED**: `version.pick` should never be invoked on module load.

File: cfme/tests/containers/test_containers_default_views.py [hunk 2]
```diff
@@ -46,6 +52,7 @@ def test_containers_providers_default_view(button_group, view):
               to Grid/Tile/List view
             * Goes to Compute --> Containers --> Providers and verifies the selected view
         """
+    view = sample(VIEWS, 1)
```
Comment (RonnyPfannschmidt) [inline cfme/tests/containers/test_containers_default_views.py]: at first glance this is a intense randomization of the collected test set

is there a good reason not to collect all of them?

Comment (gshefer) [inline cfme/tests/containers/test_containers_default_views.py]: yes - time.

**Comment by jkrocil** [comment]
This PR cannot be in [RFR] state. If you point PRT correctly at all the test cases introduced in this PR, you will discover that collection will fail.

**Comment by gshefer** [comment]
My bad - fixed.

**Pull request merged**